spec/views/posts/post_index_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe 'Render posts index page', type: :feature do
+  before :each do
+    @user = User.create(name: 'Baraka Danny', photo: 'https://unsplash.com/photos/F_-0BxGuVvo',
+                        bio: 'Dummy text for testing', post_counter: 6)
+    @first_post = Post.create(title: 'Hello', text: 'This is a post text', comments_counter: 0, likes_counter: 0,
+                              user_id: @user.id)
+    5.times do |_i|
+      Comment.create(text: 'Nice post!!', user_id: @user.id, post_id: @first_post.id)
+    end
+    visit user_post_path(@first_post.user, @first_post)
+  end
+
+  scenario 'displays post text' do
+    expect(page).to have_content('This is a post text')
+  end
+
+  scenario 'display the post title' do
+    expect(page).to have_content(@first_post.title)
+  end
+
+  scenario 'display the post body' do
+    expect(page).to have_content(@first_post.text)
+  end
+
+  scenario 'display the first comment on a post' do
+    expect(page).to have_content('Hello')
+  end
+
+  scenario 'display the how many comments' do
+    expect(page).to have_content('5')
+  end
+
+  scenario 'display the how many Likes' do
+    expect(page).to have_content('0')
```

Comment:
- [ ] please make sure to write tests for all the below senario's:
- Tests that asserts you can see the user's profile picture.
- Tests that asserts you can see the user's username.
- Tests that asserts you can see the number of posts the user has written.
- Tests that asserts you can see a post's title.
- Tests that asserts you can see some of the post's body.
- Tests that asserts you can see the first comments on a post.
- Tests that asserts you can see how many comments a post has.
- Tests that asserts you can see how many likes a post has.
- Tests that asserts you can see a section for pagination if there are more posts than fit on the view.
- Tests that redirects to the post's show page when you click on a post.

Reply:
well resolved! ✔️